recipes/joommfutil/meta.yaml

```diff
@@ -17,18 +19,22 @@ requirements:
   build:
     - python
     - pip
+    - numpy
```

Comment:
I think numpy is needed in the `run` section of the requirements - those are the things that get installed when you install this package.